src/site/includes/tags.drupal.liquid

```diff
@@ -0,0 +1,9 @@
+{% if fieldTags.entity != empty %}
+  <p>
+    <strong>Tags</strong>
+    {% for topic in fieldTags.entity.fieldTopics %}
+      <a href="{{ topic.entity.entityUrl.path }}" class="usa-button-secondary vads-u-font-size--sm vads-u-border--2px vads-u-border-color--primary-alt vads-u-padding--0p25 vads-u-margin-left--1 vads-u-text-decoration--none vads-u-color--base">{{ topic.entity.name }}</a>
```

Comment:
You have become a master of utility classes 👏

Reply:
hahaha